frontends/concrete-python/Makefile

```diff
@@ -173,7 +173,12 @@ pylint:
 
 ruff:
 	eval $(shell make silent_cp_activate)
-	ruff concrete/ examples/ scripts/ tests/ benchmarks/
+
+	ruff check concrete/ --ignore RUF010
```

Comment:
now we ignore, to not have issue with the modifications we don't like in prints

Reply:
Please add this to `.ruff.toml` not the command.

Reply:
done

Reply:
for some reasons, I have had to add 'S605 Starting a process with a shell, possible injection detected' as well